### DISCUSSION TITLE
Option to automatically split remote calls to Ray into their own tabs

### DISCUSSION THREAD

**Discussion by S-R0** [spatie/ray#451] [Ideas]
It can get very confusing and messy if you're connected to multiple servers and all of your Ray calls are together in 1 big page. It would be really handy to have a setting that automatically splits local and individual remote calls into their own tabs.

For example if i'm connected to a server (My Website Production) and leave it connected and I set ray calls live in production, if a Ray call is fired a new tab is automatically created labelled (My Website Production) with maybe a notification icon so I dont miss it.

In doing it this way I can use my Ray app normally for local development, but I can always keep my remote servers connected and wouldn't need to worry about everything getting messy, a new tab will just pop up at the top when a live error is logged with the name of the server it came from.

I know `newScreen()` is available as an option but it's not ideal having to simulate this behaviour by organising my screens and tabs in the code itself, everywhere I implement Ray.

**Comment by vesper8** [comment]
I really hope someday this will be implemented.. either with tabs, groups, or at the least ability to target specific screens after creating a new screen
